recipe/meta.yaml

```diff
@@ -32,27 +32,28 @@ requirements:
     - wheel
   run:
     - python
-    - pip    >=18.0
-    - setuptools
-    - wheel
+    - pip >=18.0
```

Comment:
I don't think pip is needed in the run section. Let me know if you have found otherwise. 
```suggestion

```